vtex-apps/minicart
# Feat/add ga4 viewcart event

#### What problem is this solving?

Implement new GA4 event `view_cart` when the minicart is opened

#### How to test it?

* Go to this [branch of minicart app](https://github.com/vtex-apps/minicart/tree/feat/add-ga4-viewcart-event)
* Link minicart app on store
* Switch to [branch with new GTM version events](https://github.com/vtex-apps/google-tag-manager/tree/feat/settings-merge-ua-events) and link on workspace
* Open the minicart and check on `window.dataLayer` if the item has the correct attributes
* In order to test minicart v1 follow [this documentation](https://github.com/vtex-apps/minicart/blob/383d7bbd3295f06d1b5854a0add561a872e1515c/docs/README.md)



#### Screenshots or example usage:

<img width="1431" alt="Captura de Tela 2023-04-05 às 18 23 09" src="https://user-images.githubusercontent.com/36740164/230214585-fd23a4ed-2d62-49a3-be45-2b6a6cf3aea5.png">


#### Describe alternatives you've considered, if any.



#### Related to / Depends on



#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/HEtutf2dvFpsqUDZFP/giphy.gif)
